delete_files_week_older.sh

```diff
@@ -0,0 +1,6 @@
+# to run as cronjob every night
+
+DAY_LAPSE=7
```

Comment:
Can this be also in environment variables and args? Would be nice if that can be controlled from env variables. 7 days can use as a default value if it isn't set.